garglk/gi_dispa.c

```diff
@@ -71,8 +66,8 @@ static gidispatch_intconst_t intconstant_table[] = {
     { "gestalt_CharOutput_ExactPrint", (2) },
     { "gestalt_DateTime", (20) },
     { "gestalt_DrawImage", (7) },
-    { "gestalt_GarglkText", (0x1100) },
```

Comment:
The garglk text stuff shouldn't have been removed.

Reply:
Yikes, good catch.  I just copied those in rather than doing a diff, completely forgetting the custom Gargoyle stuff there.